cli/src/main.rs

```diff
@@ -1204,6 +1210,19 @@ impl Highlight {
                     }
                 }
 
+                if html_mode && !quiet {
+                    println!("  <style>");
+                    let names  = theme_config.theme.highlight_names.iter();
+                    let styles = theme_config.theme.styles.iter();
+                    for (name, style) in names.zip(styles) {
+                        if let Some(css) = &style.css {
+                            println!("    .{} {{ {} }}", name, css);
```

Comment:
this is working, because the `.` of the highlight name separator works as the child class name.

This is incidental though, and will only work if the format of highlight names doesn't change, and doesn't include illegal characters for CSS class names; all of the grammars I've found work, but this feels ugly and lucky to me.

ie, not sure how better to sanitize highlight names into css classes, just letting it through for now